executor/executor.cc

```diff
@@ -573,7 +648,8 @@ static void mmap_output(int size)
 	if (output_data == NULL) {
 		if (kAddressSanitizer) {
 			// Don't use fixed address under ASAN b/c it may overlap with shadow.
```

Comment:
It says "don't use fixed address", and yet a fixed address is used just below? It looks confusing.

Reply:
Updated code/comment.